### PR TITLE
Document cluster topologies

### PR DESCRIPTION
**Pull request by chrisnegus**
*Description of changes:* This PR documents the different cluster topologies (standalone vs. management/workload) that are supported by EKS Anywhere.

### CODE REVIEW

File: docs/content/en/docs/concepts/cluster-topologies.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+An EKS Anywhere management cluster is a long-lived, on-premises Kubernetes cluster that can create and manage a fleet of EKS Anywhere workload clusters.
+The workload clusters are where customers run their applications.
+The management cluster can only be created and managed by the Amazon CLI `eksctl`.
+
+The management cluster runs on customer hardware on-premises and it does not require any connectivity back to AWS to function.
+Customers are responsible for operating the management cluster including (but not limited to) patching, upgrading, scaling, and monitoring the cluster control plane and data plane.
+ 
+## What’s the difference between a management cluster and a standalone cluster?
+From a technical point of view, they are the same.
+Regardless of which deployment topology a customer chooses, you always start by creating a singleton, standalone cluster that’s capable of managing itself.
```
Comment (elylee) [inline docs/content/en/docs/concepts/cluster-topologies.md:25]: "Regardless of which deployment topology you choose, you always start by creating..."

Comment (chrisnegus) [reply docs/content/en/docs/concepts/cluster-topologies.md]: Done.

File: docs/content/en/docs/concepts/cluster-topologies.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+## What’s the difference between a management cluster and a bootstrap cluster for EKS Anywhere?
+
+A management cluster is a long-lived entity you have to actively operate.
+The _bootstrap_ cluster is a temporary, short-lived kind cluster that is created on a separate [Administrative machine]({{< relref "clusterworkflow/#administrative-machine" >}}) to facilitate the creation of an initial standalone or management cluster.
+You do not need to interact or operate the `kind` cluster.
+The `kind` cluster will be automatically deleted by the end of the initial cluster creation.
+
+## When should I deploy a management cluster?
+If you want to run three or more EKS Anywhere clusters, we recommend them to choose a management/workload cluster deployment topology because of the advantages listed in the table below.
```
Comment (elylee) [inline docs/content/en/docs/concepts/cluster-topologies.md:44]: "If you want to run three or more EKS Anywhere clusters, we recommend you to ..."

File: docs/content/en/docs/concepts/cluster-topologies.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+
+
+## Which EKS Anywhere features support the management/workload cluster deployment topology today?
+
+|        | Supported | Notes  |
+|--------|-----------|--------|
+| Create/update/delete a workload cluster on... |||
+| <ul><li>VMware via CLI</li>  | Y | |
+| <ul><li>CloudStack via CLI</li> | Y | |
+| <ul><li>SnowGlobe via CLI</li> | Y | |
```
Comment (elylee) [inline docs/content/en/docs/concepts/cluster-topologies.md:68]: Remove the line for SnowGlobe

File: docs/content/en/docs/concepts/cluster-topologies.md [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+|        |                             |  Provides a central control plane and API to automate cluster lifecycles |
+| **Cons** |  Shared secrets such, as SSH credentials or VMware credentials, across all teams who share the cluster. |  Consumes extra resources. |
+|        |  Without a central control plane (such as a parent management cluster), it is not possible to automate cluster creation/deletion with advanced methods like GitOps or IaC. |The creation/deletion of the management cluster itself can’t be automated through GitOps or IaC. |
+|        | Circular dependencies arise if the cluster has to host a monitoring server or a local container registry. | 
+||||
+
+
+## Which EKS Anywhere features support the management/workload cluster deployment topology today?
+
+|        | Supported | Notes  |
```
Comment (elylee) [inline docs/content/en/docs/concepts/cluster-topologies.md:63]: Remote the "Notes" column. We should not publish feature ETAs

File: docs/content/en/docs/concepts/cluster-topologies.md [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+|--------|-----------|--------|
+| Create/update/delete a workload cluster on... |||
+| <ul><li>VMware via CLI</li>  | Y | |
+| <ul><li>CloudStack via CLI</li> | Y | |
+| <ul><li>SnowGlobe via CLI</li> | Y | |
+| <ul><li>Bare Metal via CLI</li> | N | ETA: 2022 Q4 |
+| Update a workload cluster on... |||
+| <ul><li>VMware via GitOps/Terraform</li> | Y ||
+| <ul><li>CloudStack via GitOps/Terraform</li> | Y ||
+| <ul><li>SnowGlobe via GitOps/Terraform</li> | N | ETA: 2022 Q4 |
```
Comment (elylee) [inline docs/content/en/docs/concepts/cluster-topologies.md:73]: Remove all mentioning of SnowGlobe

File: docs/content/en/docs/concepts/cluster-topologies.md (hunk 3 above)
Comment (chrisnegus) [reply docs/content/en/docs/concepts/cluster-topologies.md]: Done.

File: docs/content/en/docs/concepts/cluster-topologies.md (hunk 4 above)
Comment (chrisnegus) [reply docs/content/en/docs/concepts/cluster-topologies.md]: Done.

File: docs/content/en/docs/concepts/cluster-topologies.md [hunk 6]
```diff
@@ -0,0 +1,82 @@
...
+| <ul><li>Bare Metal via GitOps/Terraform</li> | N | ETA: 2022 Q4 |
+| Create/delete a workload cluster on...
+| <ul><li>VMware via GitOps/Terraform</li> | N | ETA: 2022 Q4 |
+| <ul><li>CloudStack via GitOps/Terraform</li> | N | ETA: TBD |
+| <ul><li>SnowGlobe via GitOps/Terraform</li> | N | ETA: 2022 Q4 |
```
[the remark this answers is not in the document]
Comment (chrisnegus) [reply docs/content/en/docs/concepts/cluster-topologies.md]: Done.

File: docs/content/en/docs/concepts/cluster-topologies.md [hunk 7]
```diff
@@ -0,0 +1,79 @@
...
+| Update a workload cluster on... ||
+| <ul><li>VMware via GitOps/Terraform</li> | Y |
+| <ul><li>CloudStack via GitOps/Terraform</li> | Y |
+| <ul><li>Bare Metal via GitOps/Terraform</li> | N |
+| Create/delete a workload cluster on...
+| <ul><li>VMware via GitOps/Terraform</li> | N |
+| <ul><li>CloudStack via GitOps/Terraform</li> | N |
+| <ul><li>Bare Metal via GitOps/Terraform</li> | N |
+| Install a curated package on the management cluster | Y ||
+| Install a curated package on the workload cluster from the management cluster | N |
```
Comment (elylee) [inline docs/content/en/docs/concepts/cluster-topologies.md:78]: Change this one to "Y"

File: docs/content/en/docs/concepts/cluster-topologies.md (hunk 2 above)
Comment (chrisnegus) [reply docs/content/en/docs/concepts/cluster-topologies.md]: Done.

File: docs/content/en/docs/concepts/cluster-topologies.md (hunk 5 above)
Comment (chrisnegus) [reply docs/content/en/docs/concepts/cluster-topologies.md]: Done

File: docs/content/en/docs/concepts/cluster-topologies.md (hunk 7 above)
Comment (chrisnegus) [reply docs/content/en/docs/concepts/cluster-topologies.md]: Done.

**Review by TerryHowe — APPROVED** [review]
/lgtm

**Pull request merged**